<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinDialog</class>
 <widget class="QDialog" name="WinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_header">
     <property name="text">
      <string>Ukończyłeś/-aś labirynt! Czy chcesz go zapisać?</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>Nazwa:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="le_hxw">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_difficulty">
       <property name="text">
        <string>St. trudności:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="le_difficulty"/>
     </item>
     <item row="2" column="1" alignment="Qt::AlignLeft">
      <widget class="QLineEdit" name="le_seconds"/>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_seconds">
       <property name="text">
        <string>Czas:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_hxw">
       <property name="text">
        <string>Wymiary:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="cb_ranking">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Zapisać w rankingu?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_nick">
       <property name="text">
        <string>Nick:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_nick"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Save">
       <property name="text">
        <string>Zapisz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Close">
       <property name="text">
        <string>Zamnkij</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
